juju/juju
# Rename commit command to branch and enhance

## Description

Rename commit command to branch.  Enhance renamed branch command 
to display current branch if none provided.

## QA steps

1. export JUJU_DEV_FEATURE_FLAGS=generations
2. juju bootstrap
3. juju branch                         <- Active branch is "master"
4. juju add-branch testme
3. juju branch                         <- Active branch is "testme"
3. juju branch master             <- Active branch set to "master"
3. check 'see also' of other branch related commands
4. juju help branch

## Documentation changes

This feature is still under development, however any docs referencing juju commit should be updated to use juju branch in the develop stream and enhancement noted.
